config/locales/en.yml

```diff
@@ -1043,6 +1043,10 @@ en:
       contributors_nz_html: |
         <strong>New Zealand</strong>: Contains data sourced from
         Land Information New Zealand. Crown Copyright reserved.
+      contributors_si_html: |
+        <strong>Slovenia</strong>: Contains public information of Slovenia
```

Comment:
Is there a reason not to use `data from` or `data sourced from` like other countries? The phrase `public information of` is not really standard english.